msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-meeting-stage.md

```diff
@@ -0,0 +1,196 @@
+
+Every team has a different way of communicating and collaborating tasks. To achieve these different tasks, customize Teams with apps for meetings. Enable your apps for Teams meetings and configure the apps to be available in meeting scope within their app manifest.
+
+## Tabs in Teams meetings
+
+Tabs allow the meeting participants to access services and content in a specific space within a meeting. If you're new to Microsoft Teams tab development, see [Build tabs for Teams](/microsoftteams/platform/tabs/what-are-tabs).
+
+Specifically, before creating a meeting tab, it's important to learn about the surfaces that are available to target the meeting chat view, meeting details view, meeting side panel view and meeting stage view.
+
+### Meeting details view
+
+1. In your calendar, select a meeting to which you want to add a tab.
+1. Select the **Details** tab and select :::image type="content" source="../assets/icons/add-icon.png" Border = "false" alt-text="This screenshot shows the add button on the meeting stage to add apps.":::. The app gallery appears.
+
+   :::image type="content" source="~/assets/images/apps-in-meetings/Pre-Meeting-002.png" alt-text="This screenshot shows the pre-meeting app experience in Teams meeting.":::
+
+1. In the app gallery, select the app that you want to add and follow the steps as required. After adding the tab, you can view it as.
+
+# [Desktop](#tab/desktop)
+
+   :::image type="content" source="~/assets/images/apps-in-meetings/PreMeetingTab.png" alt-text="The screenshot shows desktop Teams tabs in the meeting details view in the Teams meeting.":::
+
+# [Mobile](#tab/mobile)
+
+   :::image type="content" source="../assets/images/mobile-tab.png" alt-text="The screenshot shows mobile Teams tabs in the meeting details view in the Teams meeting.":::
+
+---
+
+### Meeting chat view
+
+1. From the Teams chat panel, select the meeting chat view.
+
+1. Select + and the app gallery appears.
+
+1. In the app gallery, select the app that you want to add and follow the steps as required. After adding the tab, you can view it as.
+
+   :::image type="content" source="../assets/images/apps-in-meetings/meeting-chat-view.png" alt-text="The screenshot shows the meeting chat view in the Teams meeting.":::
+
+### Meeting side panel view
+
+1. During a meeting, Users can add apps to the meeting using the + option from their Teams meeting window.
+
+   :::image type="content" source="../assets/images/apps-in-meetings/add-app.png" alt-text="This screenshot shows how to add an app in Teams meeting window.":::
+
+1. In the app gallery, select the app that you want to add and follow the steps as required. After adding the tab, you can view it as.
+
+   :::image type="content" source="../assets/images/side-panel-view.png" alt-text="This screenshot shows side panel view with the list of apps.":::
+
+### Meeting stage view
+
+1. Once a tab is added to the meeting side panel, you can now choose to opt into global app sharing.
```

Comment:
Developers can also enable sharing specific content of their app to stage. I believe there is work in the backlog to allow sharing to only using specific content and not requiring global app sharing